automation/recipes/meteor-qpsk.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# See: https://www.reddit.com/r/RTLSDR/comments/abn29d/automatic_meteor_m2_reception_on_linux_using_rtl/ed2teuv/
+# Meteor-M 2 satellite reception
+
+BASENAME=$1
+FREQUENCY=$2
+RECEIVE_TIMEOUT=$3
+
+NON_NORMALIZED_SIGNAL_FILENAME=${BASENAME}_non_normalized_signal.wav
+QPSK_FILENAME=${BASENAME}_qpsk.qpsk
+DUMP_PREFIX_FILENAME=${BASENAME}_dump
+PRODUCT_FILENAME_WITOUT_EXT=${BASENAME}_product
+PRODUCT_BITMAP_FILENAME=${PRODUCT_FILENAME_WITOUT_EXT}.bmp
+
+SIGNAL_FILENAME=${BASENAME}_signal.wav
+PRODUCT_FILENAME=${PRODUCT_FILENAME_WITOUT_EXT}.png
+
+# Record raw IQ data
+timeout $RECEIVE_TIMEOUT \
+    rtl_fm -M raw -f $FREQUENCY -s 288k -g 48 -p 1 | \
```

Comment:
The output of all those tools should be logged.

Reply:
Now it is logged if fail.

Reply:
Cool.